openadapt/config.py

```diff
@@ -91,7 +91,7 @@
     # Calculate and save the difference between 2 neighboring screenshots
     "SAVE_SCREENSHOT_DIFF": False,
     "SPACY_MODEL_NAME": "en_core_web_trf",
-    "SCRUB_PROVIDER_NAME": ["Presidio"],
+    "SCRUB_PROVIDER_NAME": ["presidio", "comprehend"],
```

Comment:
add comprehend to the provider list

Reply:
Can you please implement these as properties on an enum-style class? e.g.

```
class ScrubProvider:
    PRESDIO = "PRESIDIO"
    COMPREHEND = "COMPREHEND"
```

Note you can always do something like `ScrubProvider.COMPREHEND.lower()`

Reply:
[09750f3](https://github.com/OpenAdaptAI/OpenAdapt/pull/476/commits/09750f33cd49619c5dd2860065d892cac34aa0bd)

Reply:
Addressed above